src/constants/chainInfo.ts

```diff
@@ -106,19 +106,19 @@ const CHAIN_INFO: ChainInfoMap = {
     helpCenterUrl: 'https://help.uniswap.org/en/collections/3137778-uniswap-on-optimistic-ethereum-oξ',
     nativeCurrency: { name: 'Ether', symbol: 'ETH', decimals: 18 },
   },
-  [SupportedChainId.OPTIMISTIC_KOVAN]: {
+  [SupportedChainId.OPTIMISM_GOERLI]: {
     networkType: NetworkType.L2,
     blockWaitMsBeforeWarning: ms`25m`,
     bridge: 'https://app.optimism.io/bridge',
     defaultListUrl: OPTIMISM_LIST,
     docs: 'https://optimism.io/',
-    explorer: 'https://optimistic.etherscan.io/',
+    explorer: 'https://goerli-optimism.etherscan.io/',
     infoLink: 'https://info.uniswap.org/#/optimism/',
-    label: 'Optimistic Kovan',
+    label: 'Optimism Goerli',
     logoUrl: optimismLogoUrl,
     statusPage: 'https://optimism.io/status',
     helpCenterUrl: 'https://help.uniswap.org/en/collections/3137778-uniswap-on-optimistic-ethereum-oξ',
-    nativeCurrency: { name: 'Optimistic Kovan Ether', symbol: 'kovOpETH', decimals: 18 },
+    nativeCurrency: { name: 'Optimism Görli Ether', symbol: 'görOpETH', decimals: 18 },
```

Comment:
Where does this symbol come from? Why does this use ö instead of oe, like the rest of the labels?

Reply:
goerli in this same file uses the symbol https://github.com/Uniswap/widgets/pull/217/files/e43429826a446ba5a8d0cff5732e34d3418b2f33#diff-d4a719d9bb0a028a94665032a446468cdec7ba1df010b1fda5757e8b7d6aef3aL93